xcodeproj/internal/input_files.bzl

```diff
@@ -192,6 +195,8 @@ def _collect(
             # assigning to `entitlements` creates a new local variable instead
             # of assigning to the existing variable
             entitlements.append(file)
+        elif attr == automatic_target_info.exported_symbols_lists:
+            exported_symbols_lists.append(file)
```

Comment:
Not sure where I would consume these, I am seeing [this file](https://github.com/maxwellE/rules_xcodeproj/blob/2f472a114705f96d56cf838103e08b01080029f8/xcodeproj/internal/linker_input_files.bzl)

Reply:
The main question to ask is, how is bazel using this (how does the bazel command-line change in this case)? And how do we replicate that in Xcode? Once you modify the Xcode project to do that, you can work backwards from there to get it in the right place in the generation.